app/frontend/package.json

```diff
@@ -95,12 +94,15 @@
     "@types/react-test-renderer": "^16.9.3",
     "@types/redux-persist": "^4.3.1",
     "@types/webpack-env": "^1.15.3",
+    "babel-loader": "8.1.0",
```

Comment:
Did you run into this: https://github.com/storybookjs/storybook/issues/13108 ?

It's better not to have babel-loader in the explicit dependencies but if it's needed to make everything work and it is indeed working then it's fine.

Reply:
Yes, I was. See also comment below.